### PR TITLE
Moving Experimentation section

### PR DESCRIPTION
**Pull request by snehalchaudhari98**
Move the Experimentation section to a separate file, adding links to Hugging Face and FTW website in README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ This repository provides the codebase for working with the [FTW dataset](https:/
     - [Run experiments in parallel](#run-experiments-in-parallel)
   - [Inference](#inference)
     - [Sample Prediction Output (Austria Patch, Red - Fields)](#sample-prediction-output-austria-patch-red---fields)
-    - [CC-BY(or equivalent) trained models](#cc-byor-equivalent-trained-models)
+    - [CC-BY(or equivalent) trained models](#cc-byor-equivalent-trained-models) -->
```
Comment (m-mohr) [inline README.md:33]: Remove the unused part, please.

Comment (snehalchaudhari98) [reply README.md]: Yep, done.

File: README.md [hunk 2]
```diff
@@ -185,330 +173,10 @@ Explore `visualize_dataset.ipynb` to know more about the dataset.
...
-      - country 1
-      - country 2
-  dict_kwargs:
-    root: <ROOT FOLDER OF THE DATASET>
-    load_boundaries: <TRUE / FALSE WHETHER TO LOAD 3 CLASS MASKS OR NOT>
-seed_everything: <SEED VALUE>
-```
 
 # Experimentation
+For details on the experimentation process, see [Experimentation] section(./EXPERIMENTS.md).
```
Comment (m-mohr) [inline README.md:178]:
```suggestion
For details on the experimentation process, see [Experimentation section](./EXPERIMENTS.md).
```

File: README.md [hunk 3]
```diff
@@ -185,330 +173,10 @@ Explore `visualize_dataset.ipynb` to know more about the dataset.
...
-
-The script automatically detects the available GPUs and runs the specified experiments on them. Each experiment will use the configuration file specified in `experiment_configs`.
-
-```bash
-python run_experiments.py
-```
-
-The script will distribute the experiments across the specified GPUs using a queue, with each experiment running on the corresponding GPU.
-
-## Inference
```
Comment (cholmes) [inline README.md:381]: This went a little further than I was hoping - could we move this section back out? Made #46 to track.

**Pull request merged**